### DISCUSSION TITLE
How to challenge a name squatting?

### DISCUSSION THREAD

**Discussion by mcepl** [community/community#46140] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I am the lead developer of the project M2Crypto (legacy Python bindings for OpenSSL) and it is and it has been always hosted on https://gitlab.com/m2crypto/m2crypto/ since October 2015 (before that the project was hosted in the private SVN repository). Its official repository has never been hosted on GitHub (there are some unofficial mirrors).

I have been [reminded by our users](https://gitlab.com/m2crypto/m2crypto/-/issues/304) about the existence of https://github.com/m2crypto/m2crypto. I have absolutely no relationship to this repository and I am 100% certain that no developer helping with the real M2Crypto project have touched (there are not that many of us).

According to https://docs.github.com/en/site-policy/other-site-policies/github-username-policy:

> GitHub prohibits account name squatting, and account names may not be reserved or inactively held for future use. Accounts violating this name squatting policy may be removed or renamed without notice. Attempts to sell, buy, or solicit other forms of payment in exchange for account names are prohibited and may result in permanent account suspension.

So, where could I ask for this repository (and whole organization) be disbanded, removed from GitHub, and (if possible) locked, so that a repository cannot be recreated ever again?

(I have [notified](https://github.com/m2crypto/m2crypto/pull/1#issuecomment-1413672750) the owner of the repo of the violation)

**Comment by landsman** [comment]
I'm struggling with a similar thing right now. How did you resolve this issue?

**Reply by mcepl** [reply]
Seems like GitHub finally removed it, but I really do not care that much. I have meanwhile moved fully to https://gitlab.com/m2crypto/m2crypto/ and I am pretty happy there (if I have ever moved anywhere else, it would be https://git.sr.ht/~mcepl/m2crypto, never GitHub).
